ci/run.sh

```diff
@@ -3,7 +3,15 @@
 set -xeu
 set -o pipefail
 
-dchatty=1 dub test -b unittest-cov --skip-registry=all --compiler=${DC}
+# Only build the unittest binary, but don't run it
+# We want to run it ourselves to catch any bug / set timeout, etc...
+dub build -b unittest-cov -c unittest --skip-registry=all --compiler=${DC}
+
+dchatty=1
+# A run currently (2020-07-21) takes < 6 minutes on Linux
+# Try a total of three times
+timeout -s SEGV 8m ./build/agora-unittests || timeout -s SEGV 8m ./build/agora-unittests || timeout -s SEGV 8m ./build/agora-unittests
```

Comment:
gotta love bash :D

Reply:
Well the `$?` solution didn't work because we `set -e`. If we used the default behavior I could've looped.